### ISSUE TITLE
Supporting ShadowCam Image Pre-processing

### ISSUE THREAD

**Issue by volare1996** [USGS-Astrogeology/isis3#5425]
Where can I see the process of pre-processing ShadowCam using ISIS software? Thanks!

**Comment by rvwagner** [comment]
For the time being, I recommend using the "Calibrated, Map-Projected, 32-bit, COG-Formatted TIFF" file for each image (<image_name>_map_raw.tif file in the archive) _(edit: updated name to match latest tweak to the data portal interface)_.  These are 32-bit files calibrated to radiance (W/m2/sr/µm), map-projected at the native pixel scale with minimal map distortion over the image area, so they will be a good basis for any further reprojection (or for using directly). They can be imported into ISIS using GDAL: `gdal_translate -of isis3 M020633514S_map_raw.tif M020633514S.map.cub`

You won't be able to view the photometric angles for the image in ISIS, unfortunately, but for surfaces under secondary illumination, the photometry is more complicated than ISIS can calculate anyway.

**Comment by tckb** [comment]
@volare1996 the ISIS utils haven't been added yet. From my discussions with the people involved - this will be available "soon". For now, all the shadowcam images are calibrated but do  not have any "spice" info - spice init fails.  @rvwagner  hows map projected possible with no SPICE data? 🤔

**Comment by rvwagner** [comment]
We have an internal version of ISIS that has a ShadowCam camera model, but haven’t managed to get it to compatible with all the computer setups that the public version of ISIS supports.  We use that version to make the map-projected PDS files, which are what I recommend using. 

Once the files are map projected, any version of ISIS can reproject them with map2map without needing SPICE.

**Comment by tckb** [comment]
Oh I see!  I saw them but never really understood what they were. Thank you for the clarification.  So if I understand it, these are radiometric calibrated images right?

**Comment by rvwagner** [comment]
Yep.  They have the most current radiometric calibration, and are in units of W/m^2/sr/µm.  Each has a custom polar stereographic or equirectangular map projection that should minimize distortion, so you'll need to reproject them into a uniform resolution/projection if you want to mosaic them in ISIS (GIS software like ArcMap or QGIS should be able to overlay them directly).  For ISIS, use the gdal_translate command I gave above to convert them from GeoTIFF to ISIS cube.

**Comment by tckb** [comment]
I am mostly looking at SDR and Shackleton - I am trying to reconstruct the entire Shackleton so map projected data is definitely what I am looking at. But, I have never adjusted the resolution. I never had success with mosaic using ISIS so I always used QGIS/Gdal for mosaic. It worked most of the time but I saw images don't always perfectly align - is this because of the different pixel resolution?

for ref, this is what I use for 
```
Group = Mapping
  ProjectionName     = PolarStereographic
  CenterLongitude    = 0.0
  CenterLatitude     = -90.0
  TargetName         = Moon
  EquatorialRadius   = 1737400.0 <meters>
  PolarRadius        = 1737400.0 <meters>
  LatitudeType       = Planetocentric
  LongitudeDirection = PositiveEast
  LongitudeDomain    = 180
End_Group
End
```
 polar and similar in the equatorial.

**Comment by rvwagner** [comment]
The between-image offsets are expected due to the uncertainty of the spacecraft position and pointing.  Images should be within ~60m of where they're supposed to be, but can be offset randomly within that range.  If you're looking at a series of images on sequential orbits, usually they're offset from each other by a few meters per orbit in a consistent direction.

I just had a presentation at LPSC on this, and you can read the abstract here: https://www.hou.usra.edu/meetings/lpsc2024/pdf/2357.pdf

QGIS should be reprojecting everything into a uniform projection on the fly, so all you're seeing is the normal image-to-image offsets.  You'll need to use some form of reprojection/alignment tool to make a seamless mosaic; I'm not sure what QGIS has for that.

**Comment by tckb** [comment]
oh - I read your abstract but didn't realize it was you ( :) )  - interesting I did notice some inaccuracies in locations but it's good to have a number. thank you that helps - If you have the presentation of it I'd love to read it (have more questions - can I connect with you via email? ) When will ShadowCam DEM be released? - as I understand this is not released yet.  

I often use a "manual alignment plugin" to even align NAC Images - this also what I use for ShadowCam. I often wonder if there are "pre-processing" steps to perfectly match subsequent images of the ROI - for eg, featured posts on LRO blogs. Eg: https://www.youtube.com/watch?v=y3ICsoZ3Zvw ( I didn't do photometric calibration here )

**Comment by rvwagner** [comment]
Go ahead and email me directly- my email address is in that abstract.  My poster didn't have any more content than the abstract, apart from a "how to use ShadowCam data" that's the same as what I've posted here.

We expect to release ShadowCam DEMs sometime later this year, though we aren't sure of the exact timeline yet.

For alignment of NAC images, we often use deltack to align the images to a NAC DEM, and then map-project them onto the DEM.  This gets them aligned within a pixel or two (and removes any distortions from slewing), and then we do any final tweaks either in an image editor when assembling the GIF, or in ISIS by using editlab to tweak the UpperLeftCorner keywords in the Mapping group of the label.  See the "Attach camera position information->use a better shape model" part of our NAC processing guide for the map-projection part: https://www.lroc.asu.edu/files/DOCS/LROC_NAC_Processing_Guide.pdf

**Comment by tckb** [comment]
Thank you, I'll reach out. I'll followup offline.

**Comment by volare1996** [comment]
-->How soon do you plan to release DEM or camera model? Shackleton area. Thank you!

**Comment by tckb** [comment]
@volare1996  the DEM file will take some time, but you can already use LOLA for Shackleton if you are not looking for high-resolution DEMs. For example: I've combined them both to create this nice 3d view of the Shackelton crater

https://www.youtube.com/watch?v=5ksvWsXSWx8

**Comment by tckb** [comment]
hi @rvwagner  its been a while in the interest of keeping this issue alive. Any updates on this issue?

**Comment by Kelvinrr** [comment]
reopening, shadow cam is an ongoing issue.
